index.js

```diff
@@ -17,6 +17,9 @@ const lockedPorts = {
 // and a new young set for locked ports are created.
 const releaseOldLockedPortsIntervalMs = 1000 * 15;
 
+const portMin = 1024;
+const portMax = 65_535;
```

Comment:
These should be named `minPort` and `maxPort`, or maybe even `firstPort` and `lastPort`.